bark/world/evaluation/safe_distances/evaluator_static_safe_dist.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+// Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#ifndef BARK_WORLD_EVALUATION_SAFE_DISTANCES_EVALUATOR_STATIC_SAFE_DIST_
+#define BARK_WORLD_EVALUATION_SAFE_DISTANCES_EVALUATOR_STATIC_SAFE_DIST_
+
+#include "bark/world/evaluation/safe_distances/evaluator_safe_dist_base.hpp"
+
+namespace bark {
+namespace world {
+class World;
+namespace evaluation {
+
+class EvaluatorStaticSafeDist : public EvaluatorSafeDistBase {
```

Comment:
can you clarify static and dynamic?

Reply:
Static is independent of state information, just a fixed offset on front, rears and sides to increase collision shape of ego vehicle

Reply:
okay, but I would argue that "Safe Distance" indicates already that state information are used ... either we document the class with doc-strings or we could rename it to something such as EvaluatorBufferedCollisionShape?

Reply:
Mhh, but in our past valet parking projects, we always defined such terms as safe distance (or margin).